db/package.json

```diff
@@ -1,7 +1,8 @@
 {
```

Comment:
Let's use the same that we get generated from a fresh cds init --add java,sample,hana,mta with cds-dk 8:

```json
{
  "name": "deploy",
  "dependencies": {
    "hdb": "^0",
    "@sap/hdi-deploy": "^5"
  },
  "engines": {
    "node": "^18 || ^20"
  },
  "scripts": {
    "start": "node node_modules/@sap/hdi-deploy/deploy.js --use-hdb",
    "build": "npm i && npx cds build .. --for hana --production"
  }
}
```